release.sh

```diff
@@ -77,8 +79,10 @@ sed -e "s/-0.0.0/-${RELEASE_VERSION}/g" "${ROOTDIR}/nexus-repositories.yaml" \
 mkdir "${BUILDDIR}/shasta-cfg"
 "${ROOTDIR}/vendor/github.com/Cray-HPE/shasta-cfg/package/make-dist.sh" "${BUILDDIR}/shasta-cfg"
 
-# Save cray/nexus-setup and quay.io/skopeo/stable images for use in install.sh
-vendor-install-deps "$(basename "$BUILDDIR")" "${BUILDDIR}/vendor"
+# Use a newer version of cfs-config-util that hasn't rolled out to other products yet
+CFS_CONFIG_UTIL_IMAGE="arti.hpc.amslabs.hpecorp.net/csm-docker-remote/stable/cfs-config-util:5.0.0"
```

Comment:
Please move this definition into `common.sh`, together with other tools images definitions, and use

    artifactory.algol60.net/csm-docker/stable/cfs-config-util:5.0.0

instead of

    arti.hpc.amslabs.hpecorp.net/csm-docker-remote/stable/cfs-config-util:5.0.0

Network channel between our builders and arti.hpc is completely overloaded and can't provide sufficient bandwidth, we see delays and timeouts almost every time. Therefore we use copies of tools images from artifactory.algol60.net.

Reply:
I don't mind making the change, but I should point out that I just backported this from what is already in the `release/1.4` branch (and I didn't author this stuff in that branch). Because I didn't author this stuff,  have less confidence that I won't accidentally break things in making these changes. But if you feel strongly, I can take a stab at it. Do we also need to backport that to `release/1.4`?

Reply:
Yes, please.

Reply:
https://github.com/Cray-HPE/csm/pull/3325